### PR TITLE
Update DG with item feature and UCs

### PR DESCRIPTION
**Pull request by DavidTan0527**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -507,11 +684,19 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Extensions**
 
-* 2a. No items found.
+* 1a. Number of days is invalid.
+		
+    * 2a1. IBook show an error message.
```
Comment (1001mei) [inline docs/DeveloperGuide.md:713]:
```suggestion
    * 1a1. IBook show an error message.
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -507,11 +684,19 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+      Use case ends.
 
-    * 2a1. IBook shows a cute image stating nothing found.
+* 1b. No items found.
 
-#### UC9: Update all products
+    * 2b1. IBook shows a cute image stating nothing found.
```
Comment (1001mei) [inline docs/DeveloperGuide.md:719]:
```suggestion
    * 1b1. IBook shows a cute image stating nothing found.
```

**Review by MechFroG88 — APPROVED** [review]
Let's merge this for now so that others can review directly on the website

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (MechFroG88) [reply docs/DeveloperGuide.md:719]: Let's make another pr to fix these issue

**Pull request merged**